std/range/package.d

```diff
@@ -7516,7 +7516,7 @@ public:
     /// Forward range primitives. Always present.
     @property auto front()
     {
-        assert(!empty, "Attempting to access front on an empty range");
+        assert(!empty, "Attempting to access front on an empty slides");
```

Comment:
Yeah, it's awkward, but that is the name of the function that the user called and it follows the error message convention of the other structs in this package.